themes/learn2/scss/configuration/nucleus/_typography.scss

```
@@ -3,12 +3,12 @@ $core-font-size:				1.05rem !default;
 $core-line-height:				1.7 !default;
 
 // Heading Sizes
-$h1-font-size:                  $core-font-size + 2.20 !default;
-$h2-font-size:                  $core-font-size + 1.50 !default;
-$h3-font-size:                  $core-font-size + 1.10 !default;
-$h4-font-size:                  $core-font-size + 0.75 !default;
-$h5-font-size:                  $core-font-size + 0.35 !default;
-$h6-font-size:                  $core-font-size - 0.15 !default;
+$h1-font-size:                  $core-font-size + 1.5 !default;
```

```suggestion
$h1-font-size:                  $core-font-size + 1.5 !default;
```

Do we want to make these changes? Or are they as a result of the theme update? If they are custom to our theme we should be creating a child theme, otherwise we have to update this every time the Learn2 theme is updated.